tests/modules/geometry-calculation.test.js

```diff
@@ -1,9 +1,23 @@
 const Line = require('../../modules/ecs6-class/line');
 const Point = require('../../modules/ecs6-class/point');
 const geometry = require('../../modules/geometry-calculation')
-const mock = jest.fn().mockReturnValue({x: -7,y: -40})
-const mock2 = jest.fn().mockReturnValueOnce(3.5).mockReturnValueOnce(4).mockReturnValue(7)
-const mock3 = jest.fn().mockReturnValueOnce(-10.5).mockReturnValueOnce(-4).mockReturnValue(9)
+
+
+const getPointByXMock = jest
+    .spyOn(Line.prototype, 'getPointByX').mockImplementation()
+    .mockReturnValue({ x: -7, y: -40 })
+
+const calculateSlopeMock = jest
+    .spyOn(Line.prototype, 'calculateSlope').mockImplementation()
+    .mockReturnValueOnce(0).mockReturnValueOnce(7).mockReturnValue(7)
+
+
+const calculateNOfLineFunctionMock = jest
+    .spyOn(Line.prototype, 'calculateNOfLineFunction').mockImplementation()
+    .mockReturnValue(9)
+
```

Comment:
I think it's better to mock them each time again and restore
you cannot hold how many times you'll use them 
and each time you can expect a different value

Reply:
mock the class once with jest.mock, and each time mocktheReturnValue for each function